activerecord/Rakefile

```diff
@@ -207,52 +207,50 @@ end
 
 namespace :db do
   namespace :mysql do
-    mysql2_config = ARTest.config["connections"]["mysql2"]
-    mysql2_connection_arguments = lambda do |connection_name|
-      mysql2_connection = mysql2_config[connection_name]
-      ["--user=#{mysql2_connection["username"]}", ("--password=#{mysql2_connection["password"]}" if mysql2_connection["password"]), ("--host=#{mysql2_connection["host"]}" if mysql2_connection["host"]), ("--socket=#{mysql2_connection["socket"]}" if mysql2_connection["socket"])].join(" ")
+    mysql_configs = ARTest.config["connections"].slice("mysql2", "trilogy").transform_values { |connections| connections.slice("arunit", "arunit2") }
+    mysql_connection_arguments = lambda do |connection, exclude_user_password: false|
+      connection = connection.except("username", "password") if exclude_user_password
+      [
+        ("--user=#{connection["username"]}" if connection["username"]),
+        ("--password=#{connection["password"]}" if connection["password"]),
+        ("--host=#{connection["host"]}" if connection["host"]),
+        ("--port=#{connection["port"]}" if connection["port"]),
+        ("--socket=#{connection["socket"]}" if connection["socket"])
+      ].join(" ")
     end
 
-    trilogy_config = ARTest.config["connections"]["trilogy"]
-    trilogy_connection_arguments = lambda do |connection_name|
-      trilogy_connection = trilogy_config[connection_name]
-      ["--user=#{trilogy_connection["username"]}", ("--password=#{trilogy_connection["password"]}" if trilogy_connection["password"]), ("--host=#{trilogy_connection["host"]}" if trilogy_connection["host"]), ("--socket=#{trilogy_connection["socket"]}" if trilogy_connection["socket"])].join(" ")
-    end
-
-    mysql_configs = [mysql2_config, trilogy_config]
```

Comment:
`mysql2_config` and `trilogy_config` are hashes which keys contain `arunit`, `arunit2`, and more other keys.